### PR TITLE
[quant] Add op benchmark for CPU FakeQuantizePerChannel with float zero_points

### PR DESCRIPTION
**Pull request by b-koopman**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #65241

Differential Revision: [D31150087](https://our.internmc.facebook.com/intern/diff/D31150087)

### CODE REVIEW

**Review by jingsh — COMMENTED** [review]
generally looks good, except comments on the tests being commented out

File: benchmarks/operator_benchmark/pt/quantization_test.py [hunk 1]
```diff
@@ -59,11 +59,11 @@ def init(self, C, M, N, dtype, mode):
     def forward(self, input):
         return self.op(input)
 
-
+'''
 op_bench.generate_pt_test(
     quantize_per_tensor_configs_short + quantize_per_tensor_configs_long,
     QuantizePerTensorBenchmark)
-
+'''
```
[the remark this answers is not in the document]
Comment (b-koopman) [reply benchmarks/operator_benchmark/pt/quantization_test.py:66]: Thanks, will un-comment.

**Review by EdwaRen — APPROVED** [review]
lgtm!

**Review by jingsh — COMMENTED** [review]
I think there're a few more tests got commented out, uncomment those?

line 118, 167

**Review by jingsh — APPROVED** [review]
lgtm!

**Comment by b-koopman** [comment]
@b-koopman has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31150087).

**Comment by b-koopman** [comment]
@b-koopman has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31150087).

**Pull request closed without merging**